src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -119,7 +119,17 @@
   </ItemGroup>
   <ItemGroup>
     <!-- Compile tests against the System.Runtime contract, but copy our local-built implementation for testing -->
+	<ProjectReference Include="..\ref\System.Runtime.csproj">
```

Comment:
This seems to indicate we are not testing any of the other new System.Runtime APIs higher then 4.0.20?